plugin/hotswap-agent-spring-plugin/run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
On my ubuntu system, `/bin/sh` is the dash shell and does not seem to support `function test {...}`